src/locales/en-US/commands/reddit.json

```
@@ -1,15 +1,23 @@
 {
 	"name": "reddit",
 	"description": "Get information from Reddit",
+	"subredditName": "subreddit",
+	"subredditDescription": "Get a random post from a specified subreddit.",
```

No final dots in descriptions of any kind, here and everywhere.
```suggestion
	"subredditDescription": "Get a random post from a specified subreddit",
```